packages/augur-ui/src/modules/contracts/actions/contractCalls.ts

```diff
@@ -306,13 +306,12 @@ export function getDai() {
   return contracts.cashFaucet.faucet(new BigNumber('1000000000000000000000'));
 }
 
-export function fundGsnWallet(gsnWalletAddress: string) {
+export function fundGsnWallet() {
   const amount = new BigNumber('1000000000000000000000');
   const { contracts } = augurSdk.get();
 
   augurSdk.client.setUseRelay(false);
   contracts.cashFaucet.faucet(amount).then(() => {
-    contracts.cash.transfer(gsnWalletAddress, amount);
```

Comment:
this is needed for testing on kovan, when signing wallet has ETH but no DAI. this button faucets DAI and sends to GSN Wallet

Reply:
Tested, seemed to work fine without this call

Reply:
Did you point at kovan. 
paste address in kovan ETH faucet
then when the button activates click it to transfer cash to GSN wallet?